docker-compose.yml

```diff
@@ -1,13 +1,11 @@
-version: '2.0'
+version: '3'
+
 services:
   db:
-    image: postgres
+    image: postgres:alpine
```

Comment:
@diegopolido which postgres version do we use on production? then we can use the same one for development.